### ISSUE TITLE
Laragon website redirects needed

### ISSUE THREAD

**Issue by TheOldMan2000** [leokhoa/laragon#683]
HI @leokhoa 

The help links in Laragon don't work anymore because they point to the old website and need updating for convenience and for SEO when people are searching. 

Is it possible for you to update them and release a Laragon update, or could you add some quick 301 redirects on your site or via Cloudflare and point them towards the Github versions if they exist? 

For example, these are the ones that show on my Laragon, but there may be more for other users:

Under PHP Menu
How to add another PHP version:
https://forum.laragon.org/topic/166

PHP > How to manage "Quick Settings"?
https://forum.laragon.org/topic/1015/

PHP > How to add Xdebug to Laragon:
https://forum.laragon.org/topic/264

APACHE Menu
How to add another Apache version:
https://forum.laragon.org/topic/165

MariaDB Menu
How to add another MySQL version:
https://forum.laragon.org/topic/164/

NODE.js Menu
How to add another Node.js version:
https://forum.laragon.org/topic/1005

PYTHON Menu
How to add another Python version:
https://forum.laragon.org/topic/1007

TOOLS MENU
Tools > WildcardDNS
How to manage "WildcardDNS"
https://forum.laragon.org/topic/1014

Tools > How to manage "Cron"
https://forum.laragon.org/topic/2852

QUICKAPP MENU
How to manage "QuickApp"
https://forum.laragon.org/topic/1019

HTDOCS MENU > SHARE
How to share your projects with Ngrok
https://forum.laragon.org/topic/167

LARAGON MENU
LARAGON > Procfile
Procfile - What is it?
https://forum.laragon.org/topic/476

LARAGON > Usr
Usr - What is it?
https://forum.laragon.org/topic/1018

Many thanks!

**Comment by leokhoa** [comment]
Thanks @TheOldMan2000 - will do.

**Comment by TheOldMan2000** [comment]
Awesome, thank you!

**Comment by kstingel** [comment]
@leokhoa @TheOldMan2000 
have started on them, 
thanks for the list - have added to my lookups

**Comment by TheOldMan2000** [comment]
Great, thank you very much!

**Comment by alxndr-w** [comment]
@leokhoa will you change the hyperlinks inside Laragon? Suggest redirecting to their representation in the Wiki pages for all users via `301 Redirect`, which would still help Laragon 6 users.

**Comment by leokhoa** [comment]
@alxndr-w : Yes, redirecting is a solution for that. I'll redirect them to the new Laragon docs.
